data/strings/sound.txt

```diff
@@ -114,6 +115,21 @@
     zh-Hans = 向右急转。
     zh-Hant = 向右急轉。
 
+  [make_a_right_turn_street]
+    comment = Used inside formatted phrases like dist_direction_onto_street ("in 500 meters, turn right onto Main Street")
+    ja = 右折し
```

Comment:
Default en translation should be always specified.

Reply:
This will be the one exception to that rule, I meant to ask what's the safeguard for missing translations but the implemented logic is that we look for a _street translation and if it doesn't exist we fall back to the plain translation. English has no need for separate grammar inside a prepositional phrase but a few languages do.